soccer/gameplay/plays/offense/kick_gently.py

```diff
@@ -0,0 +1,96 @@
+import play
+import behavior
+import skills.move
+import skills.pivot_kick
+import constants
+import robocup
+import main
+import enum
+import timeout_behavior
+import tactics.coordinated_pass
+import evaluation.touchpass_positioning
+
+
+class KickGently(play.Play):
```

Comment:
I never got to test it with actual robots but theoretically it should work, I can do some testing with the simulator in a little bit to make sure nothing crazy happens.